src/style.css

```diff
@@ -1,3 +1,73 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  align-items: center; /* вертикально вкладені елементи центруються */
+  justify-content: space-between; /* вільне місце буде між елементами */
+  display: flex; /* елементи розташовуються горизонтально */
+  padding: 0 50px; /* додає простір навколо вкладених елементів */
+  width: 100vw;
```

Comment:
Using '100vw' for the width can cause horizontal scrolling issues on some devices due to the scrollbar width. Consider using '100%' instead to avoid this problem.